src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAnalysisController.java

```diff
@@ -719,46 +727,30 @@ public void getAllAnalysisFiles(
                     entityReference.getDataNodeId());
             throw new ServiceNotAvailableException(message);
         }
-        List<MultipartFile> files = new LinkedList<>();
         final List<ImportedFile> importedFiles = (List<ImportedFile>) responseMessage.getObject();
         boolean hasAnalysisRunner = hasAnalysisRunner(importedFiles);
         if (entityType.equals(CommonAnalysisType.ESTIMATION) && !hasAnalysisRunner) {
-            files.addAll(importService.processEstimation(importedFiles));
-        } else {
-            files = importedFiles.stream()
-                    .map(file -> conversionService.convert(file, MockMultipartFile.class))
-                    .collect(Collectors.toList());
+            return importService.processEstimation(importedFiles);
         }
+
+        List<MultipartFile> files = importedFiles.stream()
+                .map(file -> conversionService.convert(file, MockMultipartFile.class))
+                .collect(Collectors.toList());
+
         if (!hasAnalysisRunner) {
             if (entityType.equals(CommonAnalysisType.PREDICTION)) {
```

Comment:
IF can be replaced with switch(ENUM) or at least replace equals with == condition

Reply:
done